src/core/App/routes.js

```diff
@@ -0,0 +1,16 @@
+import { createHashRouter, createRoutesFromElements, Navigate, Route, } from "react-router-dom";
+import MoviesPage from "../../features/MoviesPage";
+import PersonsPage from "../../features/PersonsPage";
```

Comment:
nazwę strony też zamień na PeoplePage 😁

Reply:
A może jednak liepej będzie nazwać to jako list, a MoviesPage jako MoviesList? Bo mamy jeszcze coś takiego jak MoviDetails i PersonDetails?